lib/ansible/modules/notification/mail.py

```diff
@@ -264,8 +264,8 @@ def main():
     try:
         if secure != 'never':
             try:
-                smtp = smtplib.SMTP_SSL(host=host, timeout=timeout)
-                code, smtpmessage = smtp.connect(host, port=port)
+                smtp = smtplib.SMTP_SSL(host=host, port=port, timeout=timeout)
+                code, smtpmessage = smtp.connect(host, port)
```

Comment:
I tested the mail module on Ansible v2.7.1 on Python 3.7.1 and if you specify `secure: never` the mail module works fine. Changes are only necessary to be applied to the attempt to use ssl (which you did on lines 267 & 268). By changing this section, it breaks functionality for ansible users that run on python 2.7.x. Is there a reason you changed this section as well?